framework/doc/service_hooks.rst

```diff
@@ -14,9 +14,9 @@ application bundles.
 
 Some example use cases for service hooks include:
 
-- Proxying of existing services by hiding the original service and 
-registering a *proxy service* with the same properties 
-- Providing services *on demand* based on registered service listeners from external bundles
+   - Proxying of existing services by hiding the original service and 
+     registering a *proxy service* with the same properties 
+   - Providing services *on demand* based on registered service listeners from external bundles
 
```

Comment:
Now there is too much indentation and the block [renders as a quote](https://github.com/CSol314/CppMicroServices/blob/2cd6b067dc5a56597607fd8ae7de6bb9145a2940/framework/doc/service_hooks.rst). Do not indent the lines starting with a `-`.